### PR TITLE
feat: adjust multimodal encoder

### PR DESCRIPTION
**Pull request by bwanglzu**
1. Change TIRGMultimodalEncoder to make it work.
2. Add instance level tests for TIRGEncoder and TIRGMultiModalEncoer.

I'll add batch level test soon.

### CODE REVIEW

File: multimodal-search-tirg/TirgMultimodalEncoder /__init__.py [hunk 1]
```diff
@@ -52,25 +53,24 @@ def _get_features(self, data):
...
         text_features = self.model.extract_text_feature(textual_data)
-        
-        return self.model.compose_img_text_features(img_features, text_feature)
+
+        return self.model.compose_img_text_features(img_features, text_features)
 
     @batching
     @as_ndarray
-    def encode(self, *data: 'np.ndarray', args, **kwargs) -> 'np.ndarray':
+    def encode(self, data: list, *args, **kwargs) -> 'np.ndarray':
```
Comment (JoanFM) [inline multimodal-search-tirg/TirgMultimodalEncoder /__init__.py:70]: this signature should be:

```
def encode(self, *data: 'np.ndarray', **kwargs) -> 'np.ndarray':
```

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by JoanFM** [comment]
Why isn't here the `notebook` so that @bhavsarpratik can work on it? or the proper tests in the `executors`?

File: multimodal-search-tirg/TirgMultimodalEncoder /__init__.py (hunk 1 above)
Comment (JoanFM) [reply multimodal-search-tirg/TirgMultimodalEncoder /__init__.py:70]: the point is we added also `*args` which can't be done

Comment (bwanglzu) [reply multimodal-search-tirg/TirgMultimodalEncoder /__init__.py]: done

**Review by bhavsarpratik — APPROVED** [review]

**Pull request merged**